### DISCUSSION TITLE
Cannot delete account

### DISCUSSION THREAD

**Discussion by grikomsn** [WalletConnect/walletconnect-monorepo#3514] [Developer Support]
Seems that I cannot delete my account, whenever I click delete, this shows up: https://cloud.walletconnect.com/app/account

<img width="512" alt="CleanShot 2023-08-22 at 02 07 55@2x" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/8220954/358f129e-9a2f-48d6-b1ea-629fa1a3c357">

**Comment by bossley9** [comment]
I am also getting this error when trying to delete my account.

**Comment by glitch-txs** [comment]
Thanks for reporting, we'll look into this 🙏

**Comment by bossley9** [comment]
Hello, is there any update on this?

**Comment by bossley9** [comment]
Is there any update?
